security/pkg/pki/util/keycertbundle_test.go

```diff
@@ -371,7 +371,7 @@ func TestNewVerifiedKeyCertBundleFromFileWithCrl(t *testing.T) {
 			certChainFile: []string{crlCertChainFile},
 			rootCertFile:  crlRootCertFile,
 			crlFile:       badCrlFile,
-			expectedErr:   "missing CRL signed by certificates: [CN=Intermediate CA CN=Root CA]",
+			expectedErr:   "missing CRL signed by certificates",
```

Comment:
The order of `[CN=Intermediate CA CN=Root CA]` is indeterministic due to the map, which causes the flake. Removing it to avoid the flake.